lib/ca_certs.mli

```diff
@@ -1,5 +1,5 @@
 (* TODO delete this once https://github.com/mirage/ca-certs is on opam *)
+
+(** Detect root CAs in the operating system's trust store. Returns [None] if detection did
+    not succeed. The variants correspond to the ones used in [X509_lwt] in [ocaml-tls]. *)
```

Comment:
Does this work correctly with `odoc`? Their documentation advises to add these comments after the function.

Reply:
Yes works perfectly with odoc, it's attached to the same AST node.